streaming/src/test/scala/org/apache/spark/streaming/FailureSuite.scala

```diff
@@ -43,6 +43,9 @@ class FailureSuite extends SparkFunSuite with BeforeAndAfter with Logging {
       Utils.deleteRecursively(directory)
     }
     StreamingContext.getActive().foreach { _.stop() }
+
+    // Stop SparkContext if active
+    SparkContext.getOrCreate(new SparkConf().setMaster("local").setAppName("bla")).stop()
```

Comment:
could you explain where leaks `SparkContext`? I cannot find any clue from the log